clippy_dev/src/main.rs

```diff
@@ -180,7 +180,14 @@ fn get_clap_config() -> ArgMatches {
                         .short('n')
                         .long("name")
                         .help("Name of the new lint in snake case, ex: fn_too_long")
-                        .required(true),
+                        .required(true)
+                        .value_parser(|name: &str| {
+                            if name.contains('-') {
+                                Err("Lint name cannot contain `-`, use `_` instead.")
```

Comment:
Instead of error, would it make sense to just replace `-` with `_`?

Reply:
That makes sense to me (assuming that *was* directed at me). I think it's safe to assume that the user probably didn't want the literal character `-` but just meant to have it as a delimiter and replacing it with `_` is fine

Reply:
Yes, that question was for you :)